catalog/MDCCatalogObjectiveC/MDCCatalogObjectiveC.podspec

```diff
@@ -0,0 +1,14 @@
+Pod::Spec.new do |s|
+  s.name         = "MDCCatalogObjectiveC"
```

Comment:
Can this simply be called "MaterialCatalog"?

Reply:
Done.